protokt-codegen/src/main/kotlin/protokt/v1/codegen/generate/DeserializerGenerator.kt

```diff
@@ -31,21 +31,21 @@ import protokt.v1.UnknownFieldSet
 import protokt.v1.codegen.generate.CodeGenerator.Context
 import protokt.v1.codegen.generate.Wrapper.interceptRead
 import protokt.v1.codegen.generate.Wrapper.wrapField
-import protokt.v1.codegen.util.FieldType
-import protokt.v1.codegen.util.FieldType.Enum
-import protokt.v1.codegen.util.FieldType.SFixed32
-import protokt.v1.codegen.util.FieldType.SFixed64
-import protokt.v1.codegen.util.FieldType.SInt32
-import protokt.v1.codegen.util.FieldType.SInt64
-import protokt.v1.codegen.util.FieldType.UInt32
-import protokt.v1.codegen.util.FieldType.UInt64
 import protokt.v1.codegen.util.KotlinPlugin
 import protokt.v1.codegen.util.Message
 import protokt.v1.codegen.util.Oneof
 import protokt.v1.codegen.util.StandardField
 import protokt.v1.codegen.util.Tag
+import protokt.v1.reflect.FieldType
```

Comment:
All of these types that are used for codegen but are also going to be used in dynamic conversion are moved to shared source and stripped of any bad runtime deps, e.g. KotlinPoet. They're staying `internal` so the public API does not expose them.